src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts

```diff
@@ -348,8 +348,8 @@ export class ReplicationFormComponent implements AfterViewInit {
         type : 'textareabutton',
         name : 'repl_remote_hostkey',
         placeholder : T('Remote Hostkey'),
-        tooltip : T('Paste the host key of the remote system configured\
-                     for the Replication Task.<br>\
+        tooltip : T('Paste the host key of the destination NAS\
+                     configured for the Replication Task.<br>\
```

Comment:
Remove the "configured", it makes the sentence confusing.

Reply:
done

Reply:
Also removed the hard line break and the italics from the Scan SSH Key button.